### PR TITLE
Feature / Testimonial section

### PR DESCRIPTION
**Pull request by liemgumball**
- Create & style Testimonial section on Mobile first
image

- Style for Tablet 
image

- Style for Desktop
image

- Fix section's title font size in other sections

### CODE REVIEW

File: practice-2/src/assets/css/pages/home/sections/get-start.css [hunk 1]
```diff
@@ -120,6 +120,7 @@
     }
 
     .get-start .section__title {
+        /* align-self: stretch; */
```
Comment (huyenpham-agilityio) [inline practice-2/src/assets/css/pages/home/sections/get-start.css:123]: remove?

File: practice-2/src/assets/css/pages/home/sections/testimonial.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.testimonial {
+    display: grid;
+    grid-template-columns: 64px auto 64px;
+    grid-template-rows: auto 64px;
+    grid-template-areas:
+        "testimonial testimonial testimonial"
+        "btn-left . btn-right";
```
Comment (huyenpham-agilityio) [inline practice-2/src/assets/css/pages/home/sections/testimonial.css:7]: does it works?

Comment (liemgumball) [reply practice-2/src/assets/css/pages/home/sections/testimonial.css:7]: - Yeah, it works
image

Comment (liemgumball) [reply practice-2/src/assets/css/pages/home/sections/testimonial.css:7]: You can check it on the Netify preview deployment

**Review by huyenpham-agilityio — APPROVED** [review]

**Pull request merged**